samples/sample.js

```diff
@@ -2,34 +2,36 @@
 
 const { diag, DiagConsoleLogger, DiagLogLevel } = require('@opentelemetry/api');
 const { Resource } = require('@opentelemetry/resources');
-const { MeterProvider, PeriodicExportingMetricReader, ConsoleMetricExporter, AggregationTemporality } = require('@opentelemetry/sdk-metrics-base');
-const { DynatraceMetricExporter } = require('..');
+const { MeterProvider } = require('@opentelemetry/sdk-metrics-base');
+const { configureDynatraceMetricExport } = require("..");
 
 diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL)
 
-let exporter = new DynatraceMetricExporter({
-  prefix: 'sample', // optional
-
-  // If no OneAgent is available locally, export directly to the Dynatrace server:
-  // url: 'https://myenv123.live.dynatrace.com/api/v2/metrics/ingest',
-  // apiToken: '<load API token from secure location such as env or config file>'
-});
-
-
-// You can use the ConsoleExporter for testing locally
-// exporter = new ConsoleMetricExporter(AggregationTemporality.DELTA);
+const reader = configureDynatraceMetricExport(
+  {
+    // ExporterConfig
```

Comment:
Personally, I think it would be a little clearer if the `ExporterConfig` comment was on top of the object, right now the comment appears to be referring to the property it is above. I'd say the same about the `ReaderConfig` comment.